app/controllers/subscriptions_controller.rb

```diff
@@ -5,19 +5,21 @@ class SubscriptionsController < ApplicationController
 
   def create
     @subscription = current_user.subscriptions.build(conference_id: @conference.id)
-    if @subscription.save!
+    if @subscription.save
       redirect_to root_path, notice: "You have been subscribed to receive email notifications for #{@conference.short_title}."
     else
-      redirect_to root_path, error: subscription.errors.full_messages.to_sentence
+      redirect_to root_path, error: @subscription.errors.full_messages.to_sentence
     end
   end
 
   def destroy
     @subscription = current_user.subscriptions.find_by(conference_id: @conference.id)
+
+    redirect_to root_path, error: "You are not subscribed to #{@conference.short_title}." and return unless @subscription
```

Comment:
If you use parenthesis for redirect, you can do `&& return`, and make rubocop happy! Let me know if you understand why this happens.